src/Microsoft.AspNetCore.Routing/Template/RouteTemplate.cs

```diff
@@ -59,5 +59,24 @@ private string DebuggerToString()
         {
             return string.Join(SeparatorString, Segments.Select(s => s.DebuggerToString()));
         }
+
+        /// <summary>
+        /// Gets the parameter matching the given name.
+        /// </summary>
+        /// <param name="name">name of the parameter to match.</param>
```

Comment:
Should be "The name". Remember these are complete sentences (sometimes the subject is implied).